modulemd/v2/include/modulemd-2.0/modulemd-buildopts.h

```diff
@@ -118,6 +118,18 @@ modulemd_buildopts_remove_rpm_from_whitelist (ModulemdBuildopts *self,
                                               const gchar *rpm);
 
 
+/**
+ * modulemd_buildopts_clear_whitelist:
+ * @self: This #ModulemdBuildopts
+ *
+ * Remove all RPMs from the whitelist.
+ *
+ * Since: 2.5
+ */
+void
+modulemd_buildopts_clear_whitelist (ModulemdBuildopts *self);
```

Comment:
For naming consistency, would it be more appropriate to name the new function `modulemd_buildopts_clear_rpm_whitelist()`?

Reply:
Yeah, probably best. Thanks